### ISSUE TITLE
msgpack-1.2.8 does not load for jruby

### ISSUE THREAD

**Issue by amacleay** [msgpack/msgpack-ruby#166]
Similar to #148 I think.

Tested with jruby-9.2.5.0 on 
```
$ uname -a
Linux XXXX 4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

```
$ cd `mktemp -d`
$ rvm gemset create XXX
jruby-9.2.5.0 - #gemset created /home/amacleay/.rvm/gems/jruby-9.2.5.0@XXX
jruby-9.2.5.0 - #generating XXX wrappers.......
02:25:28 /tmp/tmp.99FxDx8vtx
$ rvm gemset use XXX
Using jruby-9.2.5.0 with gemset XXX
02:25:36 /tmp/tmp.99FxDx8vtx
$ gem i msgpack
Fetching: msgpack-1.2.8-java.gem (100%)
Successfully installed msgpack-1.2.8-java
1 gem installed
02:25:49 /tmp/tmp.99FxDx8vtx
$ jruby -rmsgpack -e 'p 1'
LoadError: no such file to load -- msgpack/msgpack
  require at org/jruby/RubyKernel.java:983
  require at /usr/share/rvm/rubies/jruby-9.2.5.0/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:59
   <main> at /home/amacleay/.rvm/gems/jruby-9.2.5.0@XXX/gems/msgpack-1.2.8-java/lib/msgpack.rb:5
  require at org/jruby/RubyKernel.java:983
   (root) at /usr/share/rvm/rubies/jruby-9.2.5.0/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1
```

Thanks!

**Comment by tagomoris** [comment]
I'm very sorry about that. `v1.2.8-java` had a problem.
I committed a change to build `-java` gem correctly: https://github.com/msgpack/msgpack-ruby/commit/4676d99bdb200da8edcfdae0a632eff7e3ae31c9
And released `v1.2.9`.

@amacleay Could you check it?

**Comment by amacleay** [comment]
Excellent work, thank you so much for your rapid fix!
